streamz/sources.py

```diff
@@ -529,8 +533,14 @@ def checkpoint_emit(_part):
 
     def start(self):
         import confluent_kafka as ck
+        if self.engine == "cudf":
+            from custreamz import kafka
```

Comment:
Is this a complete replacement for `ck`? (does it include `TopicPartition`)

Reply:
Yes, if you see the API calls like `commit`, `committed`, `get_watermark_offsets`, they're the exact same as CK, including the `TopicPartition` class. We've tried to do this deliberately to keep code duplication as minimal as possible.

Reply:
ok, why not do `from custreamz import kafka as ck` and remove the conditionals?

Reply:
Oh, I see what you're saying. A little clarification here is that the API for custreamz.kafka are exactly same as CK in the sense that they take the `TopicPartition` object(s). But, the `TopicPartition` class does not exist in custreamz.kafka.

Reply:
Deserialization happens in these API and there is not concept of `TopicPartition` since this a librdkafka C++ implementation underneath: https://github.com/jdye64/cudf/blob/7307cbe4f33a67c7d31a8d48955d755d55b03e9d/python/custreamz/custreamz/kafka.py#L72